### ISSUE TITLE
[MODULE] - Hamming distance

### ISSUE THREAD

**Issue by LeonardPuettmann** [code-kern-ai/bricks#110]
**Please describe the module you would like to add to bricks**
I want to measure the hamming distance between to embedded sentences to get an additional metric to find similar sentences.

**Do you already have an implementation?**
```python
import numpy as np
from pydantic import BaseModel
from scipy.spatial.distance import hamming
from sklearn.feature_extraction.text import TfidfVectorizer

BASE_SENTENCE = "Human gene number slashed Human genome researchers have said we have about 10,000 or 20,000 fewer genes than they originally estimated." 
YOUR_ATTRIBUTE = "headline"

def hamming_distance(record):

    text_one = BASE_SENTENCE
    text_two = record[YOUR_ATTRIBUTE].text

    tfidf = TfidfVectorizer().fit_transform([text_one, text_two])

    dense = tfidf.toarray()
    vect_one, vect_two = np.squeeze(dense[0]), np.squeeze(dense[1])

    hamming_distance = hamming(vect_one, vect_two)
    return hamming_distance
```

**Additional context**
Uses scipys hamming function.
